test/verify/check-metrics

```diff
@@ -840,64 +840,64 @@ podman save quay.io/libpod/busybox | sudo -i -u admin podman load
 
         # No matching type
         m.execute("mkdir -p /tmp/hwmon/hwmon0")
-        m.execute("echo BAT0 > /tmp/hwmon/hwmon0/name")
+        self.addCleanup(m.execute, "rm -rf /tmp/hwmon")
+        m.write("/tmp/hwmon/hwmon0/name", "BAT0")
+        m.write("/tmp/hwmon/hwmon0/temp1_input", "40000")
         m.execute("mount -o bind /tmp /sys/class")
+        self.addCleanup(m.execute, "umount /sys/class")
 
         b.wait_not_present("#current-metrics-card-cpu .temperature")
 
         # create files that contain CPU temperature
+        # ARM
         m.execute("mkdir -p /tmp/hwmon/hwmon1")
-
-        # Testing on ARM devices
-        m.execute("echo cpu_thermal > /tmp/hwmon/hwmon1/name")
-        m.execute("echo 60000 > /tmp/hwmon/hwmon1/temp1_input")
-
-        b.wait_in_text("#current-metrics-card-cpu", "60 °C")
-
-        # Testing on AMD devices
-        m.execute("echo k10temp > /tmp/hwmon/hwmon1/name")
-        m.execute("echo Tctl > /tmp/hwmon/hwmon1/temp1_label && echo Tccd1 > /tmp/hwmon/hwmon1/temp2_label && echo Tccd3 > /tmp/hwmon/hwmon1/temp3_label")
-        # setting temp1_input to the highest value as Tctl should be ignored
-        m.execute("echo 150000 > /tmp/hwmon/hwmon1/temp1_input && echo 45000 > /tmp/hwmon/hwmon1/temp2_input && echo 43000 > /tmp/hwmon/hwmon1/temp3_input")
-
+        m.write("/tmp/hwmon/hwmon1/name", "cpu_thermal")
+        m.write("/tmp/hwmon/hwmon1/temp1_input", "30000")
+        # AMD
+        m.execute("mkdir -p /tmp/hwmon/hwmon2")
+        m.write("/tmp/hwmon/hwmon2/name", "k10temp")
+        # Tctl will be ignored
+        m.write("/tmp/hwmon/hwmon2/temp1_label", "Tctl")
+        m.write("/tmp/hwmon/hwmon2/temp2_label", "Tccd1")
+        m.write("/tmp/hwmon/hwmon2/temp3_label", "Tccd3")
+        m.write("/tmp/hwmon/hwmon2/temp1_input", "40000")
+        m.write("/tmp/hwmon/hwmon2/temp2_input", "30000")
+        m.write("/tmp/hwmon/hwmon2/temp3_input", "30000")
+        # intel
+        m.execute("mkdir -p /tmp/hwmon/hwmon3")
+        m.write("/tmp/hwmon/hwmon3/name", "coretemp")
+        m.write("/tmp/hwmon/hwmon3/temp1_label", "Package id 0")
+        m.write("/tmp/hwmon/hwmon3/temp2_label", "Core 0")
+        m.write("/tmp/hwmon/hwmon3/temp1_input", "30000")
+        m.write("/tmp/hwmon/hwmon3/temp2_input", "30000")
+        # atk0110 motherboard
+        m.execute("mkdir -p /tmp/hwmon/hwmon4")
+        m.write("/tmp/hwmon/hwmon4/name", "atk0110")
+        # MB Temperature will be ignored
+        m.write("/tmp/hwmon/hwmon4/temp1_label", "MB Temperature")
+        m.write("/tmp/hwmon/hwmon4/temp2_label", "CPU Temperature")
+        m.write("/tmp/hwmon/hwmon4/temp1_input", "40000")
+        m.write("/tmp/hwmon/hwmon4/temp2_input", "30000")
+
+        b.wait_in_text("#current-metrics-card-cpu", "30 °C")
```

Comment:
You cannot tell from this whether Tctl's temp2_input was actually ignored or considered, or whether  "CPU Temperature" was detected, as they all use 30 degrees. You need to set up and test each scenario individually, and reload the page (or log out and back in, to restart the bridge).